ocaml/tests/test_clustering.ml

```diff
@@ -313,11 +313,65 @@ let test_assert_pif_prerequisites () =
 let test_assert_pif_prerequisites =
   [ "test_assert_pif_prerequisites", `Quick, test_assert_pif_prerequisites ]
 
+
+(** Test PIF.disallow_unplug is RO when clustering is enabled *)
+let check_disallow_unplug expected_value __context pif msg =
+  Alcotest.(check bool) msg
+    (Db.PIF.get_disallow_unplug ~__context ~self:pif)
+    expected_value
+
+(* Need host and network to make PIF *)
+let make_host_network_pif ~__context =
+  let host    = Test_common.make_host    ~__context () in
+  let network = Test_common.make_network ~__context () in
+  let pif     = Test_common.make_pif     ~__context ~network ~host () in
```

Comment:
We prefer not to align things because then we may have to change all the lines every time we add a new one.
This is one of the reasons we start list items with the semicolon: this way we only have to change one line when remove an item: http://elm-lang.org/docs/style-guide

Reply:
I think this would be a good addition to https://github.com/lindig/ocaml-style

Reply:
Indeed, what do you think @lindig ? I think it'd be good to mention that we shouldn't worry too much about aligning things but should keep it simple and practical, to minimise the effort when viewing diffs or changing code or rebasing.